docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -734,6 +734,19 @@ WAL retention time, in seconds, to be used for tables for which a CDC stream was
 
 Default: `14400` (4 hours)
 
+## Metric export flags
```

Comment:
Backport to stable, or is it that new?

Reply:
this is a new flags section; we don't really have a miscellaneous section so I made a new one

Reply:
Sorry I mean should this flag also be added to previous docs versions, such as stable